### DISCUSSION TITLE
Managing the standard in multiple languages

### DISCUSSION THREAD

**Discussion by robredpath** [IATI/standard-process#8] [Ideas]
In our work in the Secretariat, we regularly hear feedback that a barrier to use of the standard is the lack of materials in languages other than English. Over the last year, we've been putting in place everything we need to maintain content in multiple languages, and we're starting to roll this out for various IATI assets; those who look at Connect will have seen the launch of the Validator documentation. We're planning to translate much more content over the next year. 

Our discussions in #2 touch on some things that are important to translation, such as using clear and unambiguous language in the standard; clearer English is easier to translate as well as simply being easier to work with, regardless of your level of English. 

But, there are some other questions that come to mind as we start to think about translating the standard - I'd appreciate input on these, and others are invited to raise any other questions that they have. 

First, I'd appreciate any thoughts that people have on the role that translation should play in the process of updating the standard. Should we, for example, require that any change proposals are translated into French? Should we receive feedback in French? Or, should we run the process entirely in English and only translate the final products?

Second, I'd be interested to hear about the status of translated versions of the Standard: do changes to (for example) the Spanish version that fix a grammatical error or clarify terminology but that don't affect the English require the same change process as a change to the English? In that case, should we make more effort to solicit input from Spanish-speaking users of IATI about that change? 

Finally, it would be good to explore if different materials within the Standard should be treated differently (e.g. field definitions vs guidance). And if so, how?

**Comment by davidmegginson** [comment]
We could base our guidance on making the standard more accessible to populations in common partner countries.  That would mean prioritising French, as you mention, Spanish, and Arabic as a minimum.
